ui/src/shared/components/CustomTimeRange.js

```diff
@@ -82,6 +124,41 @@ class CustomTimeRange extends Component {
       onClose()
     }
   }
+
+  handleMovingTimeRange(movingTimeRange) {
+    return () => {
```

Comment:
the way this function appears to work, the upper bound will be fixed upon whenever the page loads and all of the `moment()` invocations happen. this means that if i stay on the dashboard and later use these shortcuts, the upper bound stays at the time when i loaded the page. i don't think this is right – seems like it should update `moment()` whenever it is `apply`ed